static/src/javascripts/projects/common/modules/ui/tabs.js

```diff
@@ -0,0 +1,104 @@
+];
+
+const getTabTarget = (tab: HTMLElement): Promise<?string> =>
+    fastdom.read(() => tab.getAttribute('href'));
+
+const hidePane = (tab: HTMLElement, pane: HTMLElement): Promise<void> => {
+    const tabList: HTMLElement = (tab.parentNode: any);
+
+    return fastdom.write(() => {
+        if (tabList) {
+            tabList.setAttribute('aria-selected', 'false');
+            NAV_CLASSES.forEach(className =>
+                tabList.classList.remove(className)
+            );
+        }
+
+        pane.classList.add('u-h');
+    });
+};
+
+const showPane = (tab: HTMLElement, pane: HTMLElement): Promise<void> => {
+    const tabList: HTMLElement = (tab.parentNode: any);
+
+    return fastdom.write(() => {
+        if (tabList) {
+            tabList.setAttribute('aria-selected', 'true');
+            NAV_CLASSES.forEach(className => tabList.classList.add(className));
+        }
+
+        pane.classList.remove('u-h', 'modern-hidden');
+        pane.focus();
+    });
+};
+
+const init = (): Promise<void> => {
+    const findTabs = (): Promise<Array<HTMLElement>> =>
+        fastdom.read(() => [...document.querySelectorAll('.tabs')]);
+
+    return findTabs().then(tabs => {
+        tabs.forEach(tab => {
+            const nav = tab.querySelector('.js-tabs');
+
+            if (!nav) {
+                return;
+            }
+
+            const { tabsInitialized } = nav.dataset || {};
+
+            if (tabsInitialized === 'true') {
+                return;
+            }
+
+            nav.addEventListener('click', (event: Event) => {
+                const target: HTMLElement = (event.target: any);
+
+                if (!target || target.nodeName !== 'A') {
+                    return;
+                }
+
+                event.preventDefault();
+
+                const currentTab = tab.querySelector('.tabs__tab--selected a');
+
+                getTabTarget(target).then(nextPaneTarget => {
+                    if (nextPaneTarget) {
+                        const nextPane = tab.querySelector(nextPaneTarget);
+
+                        if (nextPane) {
+                            showPane(target, nextPane);
+                        }
+                    }
+                });
+
+                if (currentTab) {
+                    getTabTarget(currentTab).then(currentPaneTarget => {
+                        if (currentPaneTarget) {
+                            const currentPane = tab.querySelector(
+                                currentPaneTarget
+                            );
+
+                            if (currentPane) {
+                                hidePane(currentTab, currentPane);
+                            }
+                        }
+                    });
+                }
+            });
+
+            return fastdom.write(() => {
+                nav.setAttribute('data-tabs-initialized', 'true');
+            });
+        });
+    });
+};
+
+export { init, showPane, hidePane, getTabTarget };
```

Comment:
If `showPane, hidePane, getTabTarget` are being exported for tests only can we export in a seperate `_` object to denote test only exports? I believe this is the standard we're following?

```
export const _ = {
    showPane, 
    hidePane, 
    getTabTarget
}
```